packages/almin/src/Context.ts

```diff
@@ -179,6 +179,7 @@ export class Context<T> {
             autoCommit: true
         });
         unitOfWork.open(useCaseExecutor);
+        useCaseExecutor.pipe(this._dispatcher);
```

Comment:
UseCase -> UseCaseExector -> { Unit Of Work } -> Dispatcher

Make it explicit.

- [ ] move this `pipe` in UnitOfWork?